Gemfile

```diff
@@ -0,0 +1,10 @@
+source 'https://rubygems.org'
+gem 'github-pages', group: :jekyll_plugins
+gem 'fastimage'
+gem 'img_checker'
+gem 'wdm', '>= 0.1.0' if Gem.win_platform?
+group :jekyll_plugins do
+  gem 'jekyll-gzip'
```

Comment:
We don't need jekyll plugins

Reply:
it doesn't run without these lines, i checked

Reply:
It does work! Please remove them and see for yourself.